src/toolkit.cpp

```diff
@@ -839,7 +839,7 @@ std::string Toolkit::GetMEI(const std::string &jsonOptions)
 
     if (this->GetPageCount() == 0) {
         LogWarning("No data loaded");
-        return "";
+        return {};
```

Comment:
doesn't this method return a string?

Reply:
It does.

https://stackoverflow.com/questions/26587110/returning-an-empty-string-efficient-way-in-c

Reply:
ok!

Reply:
Since there were quite a few issues with strings returned and the JavaScript binding I would prefer not to change this. Thanks.

Reply:
Do we need to generate a c_string or can we use the faster `return std::string();`?

Reply:
The speed difference is insignificant. Just leave `return "";`